SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -693,3 +681,30 @@ FVector USpatialActorChannel::GetActorSpatialPosition(AActor* Actor)
 		return FVector::ZeroVector;
 	}
 }
+
+void USpatialActorChannel::SpatialViewTick()
+{
+	if (Actor != nullptr && !Actor->IsPendingKill() && IsReadyForReplication())
+	{
+		bool bOldNetOwned = bNetOwned;
+		bNetOwned = Actor->GetNetConnection() != nullptr;
+		if (bFirstTick || bOldNetOwned != bNetOwned)
+		{
+			if (NetDriver->IsServer())
+			{
+				bool bSuccess = Sender->UpdateEntityACLs(Actor, GetEntityId());
```

Comment:
Couldn't this be done with the entity is received for the first time in `SpatialReceiver`?

Reply:
During testing I found out that changing the owner of an actor potentially closes the actor's Actor channel and opens a new one - this is to get around that to explicitly check for the first tick of a spatial actor channel. Maybe after an investigation into this problem, this could be changed